packages/validation/src/schemas.ts

```diff
@@ -26,6 +26,7 @@ export type ApplicantInformationSchemaType = z.infer<typeof applicantInformation
 export type InstitutionalRepSchemaType = z.infer<typeof institutionalRepSchema>;
 export type ProjectInformationSchemaType = z.infer<typeof projectInformationSchema>;
 export type CollaboratorsSchemaType = z.infer<typeof collaboratorsSchema>;
+export type ApplicationEditSchemaType = z.infer<typeof applicationEditSchema>;
```

Comment:
Types do not need the word `Type` in their name... they are always a type.

This is also not the type of the schema, it is the type of the object defined by the schema.

This should be named `EditApplicationRequest`. See other comment about refactor for moving this into a file dedicated to applicationRouter requests/responses.

Reply:
Also, it is confusing to have type declarations so far fro the schemas that they are inferred from. Usually I would declare the type on the line following the zod schema.